bpf/maps.go

```diff
@@ -176,7 +176,7 @@ func (b *PinnedMap) Iter(f MapIter) error {
 	args := cmd[1:]
 
 	printCommand(prog, args...)
-	output, err := exec.Command(prog, args...).CombinedOutput()
+	output, err := exec.Command(prog, args...).Output()
```

Comment:
Hit a flake here where I think there may have been some output to stderr that got mixed in with the output from Stdout.  Hence switching to `Output()`, which does also capture stderr as `err.Stderr`